src/api/config/mod.rs

```diff
@@ -153,7 +153,7 @@ impl Config {
   /// components in the encoder.
   ///
   /// If it is left unset `new_context()` will create a new default one
-  /// `new_*_channel()` will use the current global threadpool if present.
+  /// `new_*_channel()` will use the default global threadpool.
```

Comment:
With your change new_context() will use the global threadpool likewise. I can reword it myself once this lands.

Reply:
Done, WDYT?

Reply:
Thank you :)